projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -511,7 +624,7 @@ describe('AuthService', () => {
     });
 
     it('should not process the callback when query string is a sub string', (done) => {
-      window.history.replaceState(null, '', '?abccode=123&xyzstate=456');
+      window.location.search = '?abccode=123&xyzstate=456';
```

Comment:
It looks like this test needs to be adapted to work with Jest's JSDom, we set the search value on the window's location.